README.md

```diff
@@ -0,0 +1,110 @@
+# Scalar Admin k8s
+
```

Comment:
Can you add a few sentences describing what this is?
I think it's helpful even for internal developers.
Something like the following.
```
scalar-admin-k8s is a tool that creates a paused state for Scalar products (e.g., ScalarDB and ScalarDL) in a Kubernetes environment. We can use such a paused state to take backups easily and consistently across multiple diverse databases.
```
(This might need to be checked by @josh-wong if it is used.)

Reply:
I added such an explanation in 71c433ebdb7cad746e75a60eb4073d24743acc2c.

@josh-wong  
Could you please also check this new sentence? Thank you!

Reply:
I was unable to suggest changes to the sentence in a reply to this message, so I added a [new comment with my suggestion](https://github.com/scalar-labs/scalar-admin-for-kubernetes/pull/20#discussion_r1436006631).